youtube_dl/extractor/liveleak.py

```diff
@@ -87,7 +100,7 @@ def _real_extract(self, url):
             else:
                 # Maybe an embed?
                 embed_url = self._search_regex(
-                    r'<iframe[^>]+src="(http://www.prochan.com/embed\?[^"]+)"',
+                    r'<iframe[^>]+src="((?:http://www.prochan.com/embed\?)|(?:http://www.youtube.com/embed)[^"]+)"',
```

Comment:
This breaks the extraction of prochan embeds. `[^"]+` is only applied for youtube group.